queries/vouch_registry.sql

```diff
@@ -0,0 +1,109 @@
+with valid_tokens as (
+  select *
+  from erc20.tokens
+  where contract_address in (
+      '\xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB', -- COW Token
+      '\x39AA39c021dfbaE8faC545936693aC917d5E7563' -- cUSDC
+    )
+),
+recognized_bonding_pools (pool, name) as (
+  select *
+  from (
+    values
+        (replace('0x8353713b6D2F728Ed763a04B886B16aAD2b16eBD', '0x', '\x')::bytea, 'Gnosis'),
```

Comment:
Given we hardcode the bonding pools here, I wonder if we can also hardcode the owners?

Reply:
That is very clever! Then we wouldn't need any fancy logic regarding front running at all! However we may want to check their deposit was received and is substantial enough to consider them valid. I like this suggestion very much!

Reply:
What do you think about this @josojo ... Since we have to hard code the bonding pools anyway (since this contract doesn't prevent anyone from emitting these events), it wouldn't be any more inconvenient to maintain the known owners.

Reply:
Yeah, probably this is the easiest and most praticial solution

Reply:
I have updated the query to include the hard coded initial funder (aka bonding pool owner). Note that the Gnosis Pool has not yet been funded so we had to comment out this line. 

You can find a permanent version of this query at: https://dune.com/queries/668644

Now we no longer need to include valid tokens or watch for erc20 transfers.

Long term, I do think we may want to go back to the previous version before this commit but we will always be stuck having to specify the recognized pools.

At some point we may also want to monitor the pools USD balance and invalidate solvers who do not have enough insurance.